docs/Image Uploader.md

```diff
@@ -16,6 +16,16 @@ However, popular hosts like [imgur.com](https://imgur.com) and [s-ul.eu](https:/
 |Deletion link|Same as above.|
 
 ## Examples
+### ctrlv.link
+|Row|Description|
+|-|-|
+|Request URL|`https://ctrlv.link/upload.php`|
+|Form field|`img`|
+|Image link|`https://ctrlv.link/{link}`|
+|Deletion link|`https://ctrlv.link/delete.php?token={token}&id={link}`|
+
+You can also replace ctrlv.link with ctrlv.cz or ctrlv.sk to your language convenience.
+
```

Comment:
New websites have been added to the bottom in past PRs. I think we should add this to the bottom as opposed to the top.